### PR TITLE
TC-065 negative for gender

### PR DESCRIPTION
**Pull request by natdem**

### CODE REVIEW

File: test/extended/gender.js [hunk 1]
```diff
@@ -50,4 +50,17 @@ describe('Gender field suit', function () {
...
 
-});
+    describe('Negative cases', function () {
+
+        it('TC-065 No button is selected / Required', function () {
+            $(sel.name).setValue(name.default);
+            $(sel.age).setValue(age.default);
+            $(sel.storyType).click();
+            $$(sel.storyList)[story.comedy].click();
+            browser.pause(10000);
```
Comment (isenkovska) [inline test/extended/gender.js:60]: I don't think we should use browser.pause in our tests -bad practice according to Julia.

File: test/extended/gender.js [hunk 2]
```diff
@@ -50,4 +50,17 @@ describe('Gender field suit', function () {
...
+            let submitBtn = $(sel.submit).isEnabled();
+            expect(submitBtn).toEqual(false);
+        });
+    });
+
+});
```
Comment (isenkovska) [inline test/extended/gender.js:66]: Please live extra line in the end of file - good paractice.

**Review by isenkovska — CHANGES_REQUESTED** [review]
please delete browser.pause from your test and add extra line in the end of file - good practice according to Julia. Also, I don't think we should push package-lock.json into main.

**Review by ivansh83 — CHANGES_REQUESTED** [review]
Please use method /inputValues4/ from helpers to reduce code redundancy.

**Review by EAGauss — CHANGES_REQUESTED** [review]
And yeah, you have deleted the package-lock.json from the repository.
You should restore it.

**Comment by o-akhadova** [comment]
> Please use method /inputValues4/ from helpers to reduce code redundancy.

I think we should not use method /inputValues4/, because it fills in all the fields. But at this test-case we live gender field empty

**Review by EAGauss — CHANGES_REQUESTED** [review]
Nothing changed, package-lock.json is still deleted.

**Pull request closed without merging**